Assets/Scripts/NpcBehaviors/CowardNpc.cs

```diff
@@ -166,6 +170,21 @@ protected override int ChooseDialoguePath(PlayerResponse option)
         //}
         // Don't have minigame bypass
         //else
+
+        //Creates a reference to the player's inventory
+        GameObject player = GameObject.FindWithTag("Player");
+        _inventoryHolder = player.GetComponent<InventoryHolder>();
```

Comment:
The BaseNPC already has a reference to the InventoryHolder. It can be accessed with `_playerInventorySystem`.